### DISCUSSION TITLE
Alpha Tags not Working One Error

### DISCUSSION THREAD

**Discussion by austindetzel** [rtl-airband/RTLSDR-Airband#438] [Q&A]
Hello, I am streaming to broadcastify. I am trying to get frequency tags working. I get this error:

`Nov 28 12:24:14 rtl-sdr rtl_airband[495]: Failed to add shout metadata
Nov 28 12:24:18 rtl-sdr rtl_airband[495]: Failed to add shout metadata
Nov 28 12:24:51 rtl-sdr rtl_airband[495]: Failed to add shout metadata
Nov 28 12:26:03 rtl-sdr rtl_airband[495]: Failed to add shout metadata
`

But no other useful information as to what may be causing the problem. 

I am running on a raspberry pi 3 and used the install documentation from the wiki. 

In my config I have the following:

` send_scan_freq_tags = true;`

Not sure where to go from here, feedback would be appreciated.

**Comment by charlie-foxtrot** [comment]
@R1ch1e-Lee reported a similar issue #428 but was pushing to a local icecast server

**Comment by R1ch1e-Lee** [comment]
Yes I get the same error messages. My local icecast server reported bad or missing password but I am using the correct credentials with my icecast server and broadcastify but neither tags seem to work.

[2023-11-03 08:54:30] INFO admin/admin_handle_request Bad or missing password on mount modification admin request (command: metadata)
[2023-11-03 08:55:19] INFO admin/admin_handle_request Bad or missing password on mount modification admin request (command: metadata)
[2023-11-03 09:00:05] INFO admin/admin_handle_request Bad or missing password on mount modification admin request (command: metadata)
[2023-11-03 09:01:49] INFO admin/admin_handle_request Bad or missing password on mount modification admin request (command: metadata)
[2023-11-03 09:04:53] INFO admin/admin_handle_request Bad or missing password on mount modification admin request (command: metadata)

**Reply by austindetzel** [reply]
Hmm, it almost sounds like it may be server side then.

**Reply by R1ch1e-Lee** [reply]
I think it has something to do with libshout-dev version being a different version that what is needed but haven't been able to confirm that. 
My libshout-dev is version 2.4.5 and I am on version 4.2.0 of rtlsdr-airband

**Reply by austindetzel** [reply]
I think you're correct, any suggestions from contributors on how to fix this issue?

**Comment by R1ch1e-Lee** [comment]
UPDATE

My SD card died so I had to start from scratch.  On a raspberry pi 4 I installed dietpi os which is a slimmed down non gui raspberry pi OS. 
After compiling and setting rtlsdr-airband plus adding the config file it runs AND Alpha Tags are working correctly. 

Not sure what difference there could be from the installations that would fix the issue. 

I have another site that alpha tags are still not working so now I have 2 to compare and maybe figure out what the issue could be

**Comment by charlie-foxtrot** [comment]
very interesting . . . .

Did you re-install the icecast server as well or is that running on another host?

Do you know what version of `libshout` you were running before vs now?  what base OS were you using before?

**Comment by R1ch1e-Lee** [comment]
I did not reinstall the local icecast server. I had it installed earlier for testing before I went live on broadcastify.


My previous installation had an older libshout version
libshout3-dev (2.4.5-1build3)

My current installation has this version:
libshout3-dev (2.4.6-1+b1)

Hope this helps

**Reply by charlie-foxtrot** [reply]
@austindetzel what version do you have?  If an earlier one, can you try updating and rebuilding?

**Reply by R1ch1e-Lee** [reply]
RTLSDR-Airband version 4.2.0

**Reply by austindetzel** [reply]
@charlie-foxtrot I had 2.4.5, Raspbian didn't have any other versions on their apt repository so I downloaded 2.4.6 and built from that. Unfortunately, it didn't work. Possible dependency problems? I plan to do a fresh install on a new pi, using the 64bit os. I'll see if that fixes the problem.

**Reply by R1ch1e-Lee** [reply]
Did you get around the doing a fresh install and testing?

**Reply by austindetzel** [reply]
Hi, I am so sorry. I have not got around to it yet

**Reply by austindetzel** [reply]
@R1ch1e-Lee new install works! I am so sorry for the delay

**Reply by R1ch1e-Lee** [reply]
that's great.  I wonder what the issue is when a fresh install fixes it?
